### DISCUSSION TITLE
WLSH Read Prompt Data from Image Loader file format?

### DISCUSSION THREAD

**Discussion by AlphaRayReal** [Comfy-Org/ComfyUI#1406] [Q&A]
Hi :slight_smile:
Started with ComfyUI today and played around…have not a problem with the WLSH Image Reader with prompt data - there is no documentation about the file format/extension.
So actually i got only the image and have no prompts here.
I added here a Debug-Textwindow to check the prompt when it is running. I hope i’m using this right…

And second: is there a possibility to batch-read images + their prompts? I thought about a tiny node where i set a path + connect to the WLSH Image Reader. I found nothing about this. Only several entries of other searching for such a node.

![image](https://github.com/comfyanonymous/ComfyUI/assets/78987129/0710f446-f9ba-402c-a557-444919b08bc1)

**Comment by adamreading** [comment]
> Hi :slight_smile: Started with ComfyUI today and played around…have not a problem with the WLSH Image Reader with prompt data - there is no documentation about the file format/extension. So actually i got only the image and have no prompts here. I added here a Debug-Textwindow to check the prompt when it is running. I hope i’m using this right…
> 
> And second: is there a possibility to batch-read images + their prompts? I thought about a tiny node where i set a path + connect to the WLSH Image Reader. I found nothing about this. Only several entries of other searching for such a node.
> 
> ![image](https://private-user-images.githubusercontent.com/78987129/265230525-0710f446-f9ba-402c-a557-444919b08bc1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDgyNzU1NzksIm5iZiI6MTc0ODI3NTI3OSwicGF0aCI6Ii83ODk4NzEyOS8yNjUyMzA1MjUtMDcxMGY0NDYtZjliYS00MDJjLWE1NTctNDQ0OTE5YjA4YmMxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA1MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNTI2VDE2MDExOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ1ZTk5MDUwOTY2MTQ0MmE2ZmUzYjJmNDNjY2RkMmY3NWYzYThiMzJkOTUwNTA0MmUzNjcxNTFmZjM1MzU4YzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.GtL1OtKHRBNjk6ZhcCbjuKqJa5xjFY3SglHWUqp8ulk)

Did you ever find a solution - I want to batch load images from a folder - grab their prompts - and save them into an expanding csv FILE
